docs/querying/sql-window-functions.md

```diff
@@ -427,8 +427,6 @@ The number of rows considered for the `moving5` window for the `count5` column:
 
 The following are known issues with window functions:
 
--  Aggregates with ORDER BY specified are processed in the window: ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW  
-     This behavior differs from other databases that use the default of RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW.  
-     In cases where the order column is unique there is no difference between RANGE / ROWS; windows with RANGE specifications are handled as ROWS.
-- LEAD/LAG ignores the default value
-- LAST_VALUE returns the last value of the window even when you include an ORDER BY clause
+- Full range queries: RANGE x PRECEDING y FOLLOWING
+- GROUP BY queries
+- SELECT *
```

Comment:
1. Range queries only support with both bounds are either `UNBOUNDED PRECEDING / FOLLOWING` or `CURRENT ROW`. We currently do not support `RANGE x PRECEDING and y FOLLOWING`
2. Groups are not supported - `GROUPS x PRECEDING`
3. `select *` is not supported when the query contains a window function. If we plan to retrieve all columns, need to specify using column names
4. Window processors does not support unnesting MVDs. Consider Unnest or use `MV_TO_ARRAY`

Reply:
all the previously mentioned things were working differently than possibly expected by the user - and they did returned some resultset which might have been incorrect in some cases

* I think we should not list 1,2,4 as "known issues"; as they all have appropriate error messages - so they are just limitations of the current implementation
* Item 2 is not supported by Calcite either...and most people doesn't even know about it.
* We could possibly retain 3 but I think with the latest serde change that should be gone now (right?)